src/Nethermind/Nethermind.Core.Test/Encoding/BlockDecoderTests.cs

```diff
@@ -86,6 +87,18 @@ public BlockDecoderTests()
                 .WithBlobGasUsed(ulong.MaxValue)
                 .WithExcessBlobGas(ulong.MaxValue)
                 .WithMixHash(Keccak.EmptyTreeHash)
+                .TestObject,
+            Build.A.Block.WithNumber(1)
+                .WithBaseFeePerGas(1)
+                .WithTransactions(transactions)
+                .WithUncles(uncles)
+                .WithWithdrawals(8)
+                .WithBlobGasUsed(ulong.MaxValue)
+                .WithExcessBlobGas(ulong.MaxValue)
+                .WithMixHash(Keccak.EmptyTreeHash)
+                // an empty Deposit array
+                .WithDeposits(0)
+                .WithValidatorExits(new[] { new ValidatorExit(TestItem.AddressA, new byte[48]) })
```

Comment:
Considering copy-paste error in WithValidatorExists such tests shouldn't pass, if they pass it means they don't test what they should